assembly/src/errors.rs

```diff
@@ -4,6 +4,7 @@ use core::fmt;
 // ASSEMBLY ERROR
 // ================================================================================================
 
+#[derive(PartialEq)]
```

Comment:
Let's also add `Clone` and `Debug` here.

Reply:
Debug is already there. I added Clone.